src/survey-public/components/feedback/FeedbackSecondPage.tsx

```diff
@@ -39,17 +40,31 @@ export const FeedbackSecondPage: React.FC<Props> = ({
   userLocationQuestionId,
 }) => {
   const { title, description, questions, buttons } = page
+  const {
+    formState: { errors },
+    trigger,
+  } = useFormContext()
+  const watchIsMap = useFormContext().watch(`single-${isUserLocationQuestionId}`)
+  const watchIsLocationValue = useFormContext().watch(`map-${userLocationQuestionId}`)
+  const [isButtonTouched, setIsButtonTouched] = useState(false)
 
-  // watch if user choses to set a pin, update component if user choses to set a pin
-  const isMap = useFormContext().watch(`single-${isUserLocationQuestionId}`) === "1"
+  const relevantQuestions = questions!.filter(
+    (q) => ![isUserLocationQuestionId, userLocationQuestionId].includes(q.id),
+  )
+  const relevantQuestionNames = getQuestionNames(relevantQuestions)
+
+  const handleOnClick = () => {
+    setIsButtonTouched(true)
+    trigger(...relevantQuestionNames)
+  }
 
   return (
     <>
       <SurveyScreenHeader title={title.de} description={description.de} />
       {/* @ts-expect-error */}
       <Question question={questions.find((q) => q.id === isUserLocationQuestionId)} />
 
-      {isMap && (
+      {watchIsMap === "1" && (
```

Comment:
Hier mitten in der View auf `=== String(1)` zu prüfen ist unerwartet. (Kommt unten nochmal)
Kannst du vielleicht oben direkt sagen…

`const isMap = useFormContext().watch(`single-${isUserLocationQuestionId}`) === "1"`

Und darüber erklären, warum "1" hier ein String ist und sagt, dass es eine Map ist? (Was bedeutet "es ist eine map"?)